snapcraft/application.py

```diff
@@ -412,12 +411,6 @@ def _create_dispatcher(self) -> craft_cli.Dispatcher:
             default_command=commands.PackCommand,
         )
 
-    @override
```

Comment:
do we have a way to detect no longer useful overrides?

@lengau ?

Reply:
We were talking about this earlier. It could be done by running `testcraft`'s test suite and analyzing for unused code.

Reply:
Yeah the problem here isn't the override (the `@override` decorator worked as intended), but rather that I forgot to remove this from craft-application.